app/components/show/controls_component.rb

```diff
@@ -90,7 +90,7 @@ def create_text_extraction
       render ActionButton.new url: new_item_text_extraction_path(druid),
                               label: 'Text extraction',
                               open_modal: true,
-                              disabled: in_accessioning? || workflow_errors?
+                              disabled: in_accessioning? || workflow_errors? || assembling?
     end
```

Comment:
the actual change (this now includes ocrWF check in sul-dlss/dor-services-app#5083)

everything else is spec changes